src/components/info/info.tsx

```diff
@@ -27,12 +30,20 @@ const Info: React.FC = () => {
   return (
     <div className="pointer-events-auto w-full overflow-auto">
       <div className="flex flex-col items-center justify-center">
-        <div className="flex w-[100%] flex-col gap-4 px-4 py-8 sm:w-[50%]">
+        <div className="flex w-[100%] flex-col gap-4 px-0 py-8 md:w-[70%] md:px-4 lg:w-[60%] xl:w-[50%]">
           <div className="flex w-full items-center justify-between px-4">
             <h1 className="text-4xl font-bold">Info</h1>
-            <ClearIcon />
+            <a
+              href="/"
+              onClick={(e) => {
+                e.preventDefault();
+                setPathname("/");
+              }}
+            >
+              <ClearIcon />
```

Comment:
We don't need have a close button for the info and profile page. The navigation works fully through the navbar.